app/views/deploys/new.html.erb

```diff
@@ -18,6 +18,8 @@
         </div>
       <% end %>
 
+      <%= Samson::Hooks.render_views(:before_deploy_form, self, project: @project, form: form) %>
```

Comment:
Actually one hook is good enough already but I think it will be nicer if we can make the field on top or below the revision field. It just looks nicer when the order is more natural imho. Should I remove `before` and keep `after` only?